src/viam/sdk/common/proto_type.hpp

```diff
@@ -30,13 +62,13 @@ class ProtoType {
     friend bool operator==(const ProtoType& lhs, const ProtoType& rhs);
 
     template <typename T>
-    T* get() {
-        return boost::get<T>(&proto_type_);
+    auto get() {
```

Comment:
Potentially this could be `decltype(auto)` but in practice I don't think it matters here.